tests/test_users.py

```diff
@@ -236,3 +236,44 @@ def test_get_user_permissions(self, client):
         role = client.get_role_by_code('admin')
 
         assert len(permissions) == len(role.roleperms)
+
+    def test_user_has_failed_login_count(self, client, user_json_object):
```

Comment:
Have these tests all passed using the --remote-connection option? (i.e. using hydra-server)